katib/installs/katib-external-db/secrets.env

```diff
@@ -0,0 +1,4 @@
+KATIB_MYSQL_DB_DATABASE=
+KATIB_MYSQL_DB_HOST=
```

Comment:
Should you also add `KATB_MYSQL_DB_PORT` here?

Reply:
Added and fixed the typo